### ISSUE TITLE
Add option to exclude a runner

### ISSUE THREAD

**Issue by julianrubisch** [prontolabs/pronto#365]
This requires some explanation. IIUC you can specify which runners to run with `-r`, but pronto will actually use this to `require` a certain runner.

I wrote a `ruby-standardrb` runner https://github.com/julianrubisch/pronto-standardrb (see respective PR #364) basically only by subclassing `pronto-rubocop` (because that's what `standard` is, an opinionated wrapper around rubocop).

Now of course in that implementation I need to `require` the rubocop runner: https://github.com/julianrubisch/pronto-standardrb/blob/c5669db79609f4fba40ba04362fe37130ea0f930/lib/pronto/standardrb.rb#L3

which will also load (and thus, run) it and lead to conflicting outputs since the base rubocop configuration is quite different from `standard`. 

so I could basically copy-paste everything from `pronto-rubocop` and not require it at all, but being all tuned to DRY implementations this really makes me want to tear my hair out...

---

TL;DR: Runners cannot be easily subclassed.

**Comment by ashkulz** [comment]
@julianrubisch can you check if the linked PR (_which is now merged_) solves your problem?

**Comment by julianrubisch** [comment]
does look good!
